rockusb/examples/rockusb.rs

```diff
@@ -29,6 +30,12 @@ fn read_flash_info(mut transport: Transport) -> Result<()> {
     Ok(())
 }
 
+fn device_reset(mut transport: Transport) -> Result<()> {
+    // TODO allow users to choose their own reset method
+    transport.device_reset(ResetMethod::Reset)?;
```

Comment:
seems pretty straightforward to sort out that todo right away :)

Reply:
had some problems exposing the enum to clap, hopefully i did it right ;-)